DP02_17_Galaxy_Photometry.ipynb

```diff
@@ -0,0 +1,1077 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        """Wrapper function to generate a cutout using the cutout tool
</pre><p>See the example in <a href="https://github.com/lsst/rtn-045/blob/main/_static/template.ipynb" target="_blank">https://github.com/lsst/rtn-045/blob/main/_static/template.ipynb</a></p><p>New line after """</p><p>Align the ----- with the text</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/231/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>done!</p>